lib/engine/round/g_18_al/stock.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require_relative '../stock'
+
+module Engine
+  module Round
+    module G18AL
+      class Stock < Stock
+        def initialize(entities, game:, **opts)
+          super
+          @bonus_controller = player_controlling_route_bonus_ability
```

Comment:
this could just be route_bonus_ability&.owner&.owner

Reply:
Fixed